Bottom: 7f01eb6134abbcdba4e1acc8f545f403248e9e3b
Top:    fdcb00412538dd83860355a75f0fbcc27fb3bf50
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-04-26 16:06:59 +0100

Refresh of add-stepcode-based-assembly

---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 95fd193..3e863de 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -927,25 +927,25 @@ step_do_export (HID_Attr_Val * options)
             attribute = AttributeGet (element, "PCB::3d_model::origin::X");
             g_string_printf (value, "%s", attribute != NULL ? attribute : "0.0 mm");
             attribute = AttributeGet (element, "PCB::3d_model::origin::Y");
-            g_string_append_printf (value, "%s", attribute != NULL ? attribute : "0.0 mm");
+            g_string_append_printf (value, " %s", attribute != NULL ? attribute : "0.0 mm");
             attribute = AttributeGet (element, "PCB::3d_model::origin::Z");
-            g_string_append_printf (value, "%s", attribute != NULL ? attribute : "0.0 mm");
+            g_string_append_printf (value, " %s", attribute != NULL ? attribute : "0.0 mm");
             AttributePutToList (&element->Attributes, "PCB::3d_model::origin", value->str, true);
 
             attribute = AttributeGet (element, "PCB::3d_model::axis::X");
             g_string_printf (value, "%s", attribute != NULL ? attribute : "0.0");
             attribute = AttributeGet (element, "PCB::3d_model::axis::Y");
-            g_string_append_printf (value, "%s", attribute != NULL ? attribute : "0.0");
+            g_string_append_printf (value, " %s", attribute != NULL ? attribute : "0.0");
             attribute = AttributeGet (element, "PCB::3d_model::axis::Z");
-            g_string_append_printf (value, "%s", attribute != NULL ? attribute : "0.0");
+            g_string_append_printf (value, " %s", attribute != NULL ? attribute : "0.0");
             AttributePutToList (&element->Attributes, "PCB::3d_model::axis", value->str, true);
 
             attribute = AttributeGet (element, "PCB::3d_model::ref_dir::X");
             g_string_printf (value, "%s", attribute != NULL ? attribute : "0.0");
             attribute = AttributeGet (element, "PCB::3d_model::ref_dir::Y");
-            g_string_append_printf (value, "%s", attribute != NULL ? attribute : "0.0");
+            g_string_append_printf (value, " %s", attribute != NULL ? attribute : "0.0");
             attribute = AttributeGet (element, "PCB::3d_model::ref_dir::Z");
-            g_string_append_printf (value, "%s", attribute != NULL ? attribute : "0.0");
+            g_string_append_printf (value, " %s", attribute != NULL ? attribute : "0.0");
             AttributePutToList (&element->Attributes, "PCB::3d_model::ref_dir", value->str, true);
 
             g_string_free (value, true);
